vehicle/autoware_external_cmd_converter/src/node.cpp

```diff
@@ -147,6 +148,14 @@ double ExternalCmdConverterNode::calculate_acc(const ExternalControlCommand & cm
 {
   const double desired_throttle = cmd.control.throttle;
   const double desired_brake = cmd.control.brake;
+  if (
+    std::isnan(desired_throttle) || std::isnan(desired_brake) || std::isinf(desired_throttle) ||
+    std::isinf(desired_brake)) {
+    std::cerr << "Input brake or throttle is out of range. returning 0.0 acceleration."
+              << std::endl;
+    return 0.0;
+  }
+
```

Comment:
```suggestion
  if (
    std::isnan(desired_throttle) || std::isnan(desired_brake) || std::isinf(desired_throttle) ||
    std::isinf(desired_brake)) {
    std::cerr << "Input brake or throttle is out of range. returning 0.0 acceleration."
              << std::endl;
    return 0.0;
  }

```
@TakaHoribe I noticed there is no proper handling of Nan or infinite values in the external_cmd_converter so I decided to add this check, but I am wondering if returning 0 acceleration is the right answer in this case? 

I have also made checks to combinations of infinity and nan inputs and checking the output to not be NaN or inf.